### PR TITLE
All maps generated from the tool + HQ assets

### PR DESCRIPTION
**Pull request by tomaas-zeman**
en-11
en-13
en-18
en-20
en-28
en-32
en-36

### CODE REVIEW

File: clash/bloody_grail.tex [hunk 1]
```diff
@@ -123,8 +123,8 @@ \subsection*{\MakeUppercase{Additional Rules}}
 
 \begin{tikzpicture}[overlay]
   \centering
-  \node at (4.5, -5) {\includegraphics[scale=0.45]{\_assets/maps/bloody-grail-2p.png}};
+  \node at (4.5, -5) {\includegraphics[scale=0.165]{\_assets/maps/bloody-grail-2p.png}};
```
Comment (qwrtln) [inline clash/bloody_grail.tex:126]: This is an example where you need only 20% of the actual size of the image.

Comment (qwrtln) [reply clash/bloody_grail.tex:126]: But ghostscript reduces the file size from 73 to 15 MB :O

Comment (qwrtln) [reply clash/bloody_grail.tex:126]: So I guess we're safe with those sizes for now.

Comment (tomaas-zeman) [reply clash/bloody_grail.tex:126]: The problem with compressing images to 20%, or any other smaller size, is that if we suddenly decide to make it larger, we need to recreate it from scratch.

On the other hand, these large images are quite heavy even on pdf browsing. I noticed slower rendering on M1 Mac.

So in the end I will probably resize them a bit but maybe to like 75% of the current size or so.

**Review by qwrtln — APPROVED** [review]

**Pull request merged**